### PR TITLE
Update nginx to v1.11.5

### PR DESCRIPTION
**Pull request by hfm**

### CODE REVIEW

File: src/stream/ngx_stream_mruby_module.c [hunk 1]
```diff
@@ -699,7 +699,18 @@ static ngx_int_t ngx_stream_mruby_init(ngx_conf_t *cf)
 {
   ngx_stream_core_main_conf_t *cmcf = ngx_stream_conf_get_module_main_conf(cf, ngx_stream_core_module);
 
+#if (nginx_version < 1011005)
```
Comment (matsumoto-r) [inline src/stream/ngx_stream_mruby_module.c]: Generally, `hoge >= 1011005` was often used

Comment (hfm) [inline src/stream/ngx_stream_mruby_module.c]: Thanks. I fixed it by a7fa1e8 .

**Comment by hfm** [comment]
Ok, CI passed 😄

**Comment by matsumoto-r** [comment]
Thanks!

**Pull request merged**